GenericJSONTests/QueryingTests.swift

```diff
@@ -58,4 +58,27 @@ class QueryingTests: XCTestCase {
         XCTAssertEqual(json.foo, JSON.string("bar"))
         XCTAssertEqual(json.nonesuch, nil)
     }
+    
+    func testKeyPath() {
+        let json: JSON = [
+            "string": "foo bar",
+            "boolean": true,
+            "number": 123,
+            "object": [
+                "str": "col",
+                "arr": [1, 2, 3],
+                "obj": [
+                    "x": "rah",
+                    "y": "tar",
+                    "z": "yaz"
+                ]
+            ]
+        ]
+        XCTAssertEqual(json[keyPath: KeyPath("string")], JSON.string("foo bar"))
```

Comment:
By the way, it’s possible to leave out the `KeyPath` type here:

    XCTAssertEqual(json[keyPath: "string"], JSON.string("foo bar"))

This is the whole point of the `ExpressibleByStringLiteral` conformance.

Reply:
Thanks 😃